src/sage/interfaces/expect.py

```diff
@@ -640,17 +643,17 @@ def quit(self, verbose=False):
 
         EXAMPLES::
 
-            sage: a = maxima('y')
-            sage: maxima.quit(verbose=True)
-            Exiting Maxima with PID ... running ...maxima...
+            sage: a = gap('(1,2)(3,7)(4,6)(5,8)')
+            sage: gap.quit(verbose=True)
+            Exiting Gap with PID ... running ...gap...
```

Comment:
maybe an easier modification of this doctest is `from <correct path> import maxima`. what if later gap pexpect get deprecated too?

Reply:
Perhaps, given that the old pexpect maxima interface is now restored again. It shouldn't be essential what interface is used here. Thus, I’d lean toward keeping it as is to avoid unnecessary changes.